modules/integration/tests-common/admin-clients/src/main/java/org/wso2/identity/integration/common/clients/registry/PropertiesAdminServiceClient.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Change the year

Reply:
Commit year is 2017. So this should be correct.